packages/velo-external-db-core/lib/schema_hooks_utils.js

```diff
@@ -0,0 +1,68 @@
+
+const hooksForAction = (action) => {
+    const hooksForAction = {
+        beforeList: ['beforeAll', 'beforeRead', 'beforeList'],
+        afterList: ['afterAll', 'afterRead', 'afterList'],
+        beforeListHeaders: ['beforeAll', 'beforeRead', 'beforeListHeaders'],
+        afterListHeaders: ['afterAll', 'afterRead', 'afterListHeaders'],
+        beforeFind: ['beforeAll', 'beforeRead', 'beforeFind'],
+        afterFind: ['afterAll', 'afterRead', 'afterFind'],
+        beforeCreate: ['beforeAll', 'beforeWrite', 'beforeCreate'],
+        afterCreate: ['afterAll', 'afterWrite', 'afterCreate'],
+        beforeColumnAdd: ['beforeAll', 'beforeWrite', 'beforeColumnAdd'],
+        afterColumnAdd: ['afterAll', 'afterWrite', 'afterColumnAdd'],
+        beforeColumnRemove: ['beforeAll', 'beforeWrite', 'beforeColumnRemove'],
+        afterColumnRemove: ['afterAll', 'afterWrite', 'afterColumnRemove']
+    }
+    return hooksForAction[action]
+}
+
+const Operations = {
```

Comment:
wdyt about creating a `consts.js` file with all the consts, and importing them when needed?